Content.Shared/Destructible/SharedDestructibleSystem.cs

```diff
@@ -28,7 +30,7 @@ public void BreakEntity(EntityUid owner)
 /// </summary>
 public sealed class DestructionEventArgs : EntityEventArgs
 {
-
+    public EntityUid? Gatherer { get; set; }
 }
```

Comment:
This shouldn't have the gatherer entityuid here as all the other usages don't care about it / it's not possible to get, just revert this change and I'll clean it up later.